README.md

```diff
@@ -63,6 +63,8 @@ Flags:
       --heartbeat-enable                         Forward HTTP heartbeat event
       --heartbeat-times strings                  Times of day to send heartbeat (list of 24h HH:MM strings)
   -h, --help                                     help for run
+      --in-buffer uint                           input buffer length (counted in EVE objects) (default 500000)
```

Comment:
maybe --in-buffer-length would be more consistent

Reply:
Good point.